ludwig/schema/features/loss/loss.py

```diff
@@ -368,10 +374,13 @@ class SigmoidCrossEntropyLossConfig(BaseLossConfig):
         description="Type of loss.",
     )
 
-    class_weights: Union[List[float], float, None] = schema_utils.List(
-        list_type=float,
+    class_weights: Union[List[float], float, dict, None] = schema_utils.OneOfOptionsField(
```

Comment:
nit: type annotation does not match the field options

Reply:
Good catch!